ext/kernel/filter.c

```diff
@@ -218,7 +218,7 @@ void zephir_escape_multi(zval *return_value, zval *param, const char *escape_cha
 		/**
 		 * Alphanumeric characters are not escaped
 		 */
-		if (value < 123 && isalnum(value)) {
+		if (value < 256 && isalnum(value)) {
```

Comment:
@ruudboon Is this correct, talking about the work you did with multibyte on macOS

Reply:
I didn't change manually this file, it happends when I compile Phalcon with Zephir.
I will restore the file like it is in the v4.0 branch.